packages/react/src/components/accordion/index.tsx

```diff
@@ -0,0 +1,2 @@
+export { Accordion } from './accordion';
+export { AccordionContainer } from './accordion-container';
```

Comment:
Naming-wise, shouldn't the `AccordionContainer` be named `Accordion`, and the `Accordion` a `Collapsible`?

Reply:
do we have a name convention guide of some sort?

Reply:
I refactor the code to get the disabled and default expanded implemented per accordion item.
the code is a bit different now.

Reply:
ok renaming to Accordion/AccordionItem

Reply:
Name refactor is done. 
Now the Wrapper is <Accordion>
the children is <AccordionItem>